src/main/java/duke/exec/Add.java

```diff
@@ -0,0 +1,39 @@
+package duke.exec;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class Add implements Executable {
```

Comment:
I think can add javadoc to explain what the class does and perhaps change the name to more specific like "AddTask", I noticed the same issue in several other places too.